x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/steps/install_fleet_server.tsx

```diff
@@ -84,7 +84,21 @@ const InstallFleetServerStepContent: React.FunctionComponent<{
       <EuiText>
         <FormattedMessage
           id="xpack.fleet.fleetServerFlyout.installFleetServerInstructions"
-          defaultMessage="Install Fleet Server agent on a centralized host so that other hosts you wish to monitor can connect to it. In production, we recommend using one or more dedicated hosts. "
+          defaultMessage="Install Fleet Server agent on a centralized host so that other hosts you wish to monitor can connect to it. In production, we recommend using one or more dedicated hosts. For additional guidance, see our {installationLink}."
+          values={{
+            installationLink: (
+              <EuiLink
+                target="_blank"
+                external
+                href="https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html"
```

Comment:
Should we use the doclink service instead of an hardcoded link?

Reply:
Swapped to doclinks in https://github.com/elastic/kibana/pull/132271/commits/2a7e7963f9a32ff457d7a110ea393eb4e5e46762